### DISCUSSION TITLE
Gradle track Preview is out 🥳

### DISCUSSION THREAD

**Discussion by y9vad9** [y9vad9/kotlin-course#78] [Announcements]
Hello, everyone 🤗
Recently, I've finished [Gradle track preview](https://course.y9vad9.com/en/docs/gradle/intro) (only in Ukrainian). 

Also, there is some little improvements:
  - Design changes: icon for navbar, new purple accent color and collapsing navbar (special thanks to @kotleni)
  - Main page updated: more information about author, courses and blog.

![image](https://user-images.githubusercontent.com/32961194/203146745-8d53791b-2809-46df-a94f-57ba3fedb500.png)
![image](https://user-images.githubusercontent.com/32961194/203146667-308f41f4-a188-4862-9f1b-69e090a0519e.png)

**Comment by kotleni** [comment]
It's great news!
